Docfile.yml

```diff
@@ -13,7 +13,7 @@ content_map:
 -
   directory: src/mftf/v2
   repository: magento/magento2-functional-testing-framework
-  branch: 2.6.5
+  branch: 2.x-develop
```

Comment:
This change doesn't belong here, does it?

Reply:
MFTF releases versions by tag. We pull the v2 MFTF from this tag. But, they won't take changes to a tag.
Since they are on to version 3, 2.x-develop is the only real branch I can do doc updates to. This is, at least to me, a new wrinkle in the multi-repo world.